packages/menu/src/Menu.ts

```diff
@@ -72,6 +72,51 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
 
     protected rovingTabindexController?: RovingTabindexController<MenuItem>;
 
+    /**
+     * iPad scroll detection properties
+     *
+     * This feature prevents menu item selection during iPad scrolling to avoid
+     * accidental selections when users are trying to scroll through a long menu.
+     *
+     * How it works:
+     * 1. On touchstart: Record initial Y position and timestamp
+     * 2. On touchmove: Calculate vertical movement and time elapsed
+     * 3. If movement > threshold AND time < threshold: Mark as scrolling
+     * 4. On touchend: Reset scrolling state after a delay
+     * 5. During selection: Prevent selection if scrolling is detected
+     *
+     * This prevents the common iPad issue where users accidentally select menu
+     * items while trying to scroll through the menu content.
+     */
+    private touchStartY = 0;
+    private touchStartTime = 0;
+    private isScrolling = false;
+
+    /**
+     * Minimum vertical movement (in pixels) required to trigger scrolling detection.
+     * Small movements (like tapping) won't trigger scrolling, but intentional
+     * scroll gestures will.
+     */
+    private scrollThreshold = 10; // pixels
+
+    /**
+     * Maximum time (in milliseconds) for a movement to be considered scrolling.
+     * Quick movements within this timeframe are likely intentional scrolls,
+     * while slower movements are more likely taps or selections.
+     */
+    private scrollTimeThreshold = 300; // milliseconds
+
+    /**
+     * Public getter/setter for scrolling state
+     */
+    public get scrolling(): boolean {
```

Comment:
I think the naming here is off. This should be the `isScrolling` API for customers and we don't necessarily need a duplicative static `isScrolling` boolean if we use the logic I proposed above to check if the start variables are undefined.